### ISSUE TITLE
Ganeti does not handle "migrating-" instances gracefully

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#1073]
Originally reported of Google Code with ID 1017.

```
<b>What steps will reproduce the problem?</b>
1. Start a couple of instance migrations on a cluster (with xen).
2. Run a ganeti operation like gnt-instance list (or the equivalent in rapi).
3. You will get this lovely exception:

Traceback (most recent call last):
[...]
  File "/my/rapi/client/code/rapi_client.py", line 559, in _SendRequest
    raise GanetiApiError(msg, code=http_code)
rapi_client.GanetiApiError: 502 Caught exception: 'migrating-nebel8.mtv.corp.google.com': Invalid responses from another server/proxy.

```
Originally added on 2015-01-13 13:19:20 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
While I agree this is an unpleasant bug that has been unreported for a surprisingly long time, I won't make this fix on 2.10. It can't be resolved without changing the hypervisor interface, with safe but far-reaching refactoring needed.

As a compromise, I'll fix this issue in 2.12 - stable, but supported for a longer period of time.
```
Originally added on 2015-01-15 12:29:30 +0000 UTC.

Added Labels: Hypervisor-XenPVM
Added to Milestone: Release2.12
Removed from Milestone: Release2.10

**Comment by Ganeti-Issues-Migrator** [comment]
```
I have tested this on the latest version of Ganeti and I was not able to replicate the issue. I have ran multiple parallel instance migrations and I was able to successfully query instance state (gnt-instance info / gnt-instance list) without any crashes. 

I'll be closing this issue, feel free to re-open it if you think it's still applicable and have a clearer amount of steps to replicate it.
```
Originally added on 2016-12-20 11:53:18 +0000 UTC.

Changed State: NotRepeatable
Removed from Milestone: Release2.12
